### PR TITLE
Leverage serialized engine to speed-up CLI

### PR DESCRIPTION
**Pull request by remusao**
Before (milliseconds): init: 3, loading: 143, matching: 1, total: 147
After (milliseconds): init: 2, loading: 7, matching: 2, total: 11

Also ran `npm audit fix`

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -1,46 +1,68 @@
...
+  // Generate binary engine if missing
+  const trackerEnginePath = path.join(
+    BASE_PATH,
+    'dist',
+    `trackerdb_${ENGINE_VERSION}.engine`,
+  );
+  if (!existsSync(trackerEnginePath)) {
+    const rawTrackerDB = JSON.parse(readFileSync(trackerDBPath, 'utf-8'));
+    const engine = FiltersEngine.fromTrackerDB(rawTrackerDB);
+    writeFileSync(trackerEnginePath, engine.serialize());
```
Comment (chrmod) [inline cli.js:35]: can we replace this with 
`await import('./scripts/export-engine/index.js');` maybe?

**Review by chrmod — CHANGES_REQUESTED** [review]

**Review by chrmod — APPROVED** [review]

**Comment by chrmod** [comment]
Thank you @remusao !

File: cli.js (hunk 1 above)
Comment (remusao) [reply cli.js:35]: Maybe. I am not sure if the behavior of `export-engine/index.js` is equivalent but it's worth checking if it simplifies the logic.

**Review by Bankrollshow — COMMENTED** [review]
- [x] - **** __ []() image@

**Pull request merged**